src/Zcu/PerThread.zig

```diff
@@ -3069,14 +3075,6 @@ pub fn structPackedFieldBitOffset(
     unreachable; // index out of bounds
 }
 
-pub fn getBuiltin(pt: Zcu.PerThread, name: []const u8) Allocator.Error!Air.Inst.Ref {
-    const zcu = pt.zcu;
-    const ip = &zcu.intern_pool;
-    const nav = try pt.getBuiltinNav(name);
-    pt.ensureCauAnalyzed(ip.getNav(nav).analysis_owner.unwrap().?) catch @panic("std.builtin is corrupt");
-    return Air.internedToRef(ip.getNav(nav).status.resolved.val);
-}
-
```

Comment:
Why were these moved into Sema? I moved them here intentionally, because we don't need the parent analysis for this. I feel like a panic is better than a "proper" compile error if something in `std.builtin` fails analysis; such an error means the standard library is corrupt, which the user probably did unintentionally, so should be told about directly. (The panic should, admittedly, have a more helpful message.)

Reply:
Because the user can get the interface wrong, and need to know about it. Furthermore,

> One thing that's neat about this approach is that you can omit any given panic function and receive a compile error if a call to that function would have been generated. For compiler-rt we pass empty struct (when not in test mode)!

 this means that you want to see the context for why a particular call is being emitted.

Proper compiler error is absolutely required since the user now controls these functions.

Reply:
Ah, I see, Sorry, I missed that -- haven't looked too closely at the changes here yet. This seems fine then.